py/tests/unit/with_runtime_sparkling/test_drf.py

```diff
@@ -60,7 +60,8 @@ def testCalibrationDataFrameCauseGenerationOfCalibratedProbabilities(prostateDat
                   seed=1,
                   splitRatio=0.8,
                   calibrateModel=True,
-                  calibrationDataFrame=calibrationDataset)
+                  calibrationDataFrame=calibrationDataset,
+                  withDetailedPredictionCol=True)
```

Comment:
qq: should we completely remove this parameter since it is not used?

Reply:
I've added this parameter back to test it doesn't cause any braking change. By manual testing, I've noticed that init method throws exception if `withDetailedPredictionCol=True` is there. I will remove the parameter completely in 3.34.